maintainers/working_groups/learning_journey/documentation-structure.md

```diff
@@ -0,0 +1,24 @@
+- Tutorials
+    - Nix
+    - NixOS
```

Comment:
This needs much more work on breaking down which skills people need to use the eco system effectively. Priming the group to think in terms of projects seems counterproductive. Tutorials and guides are that part where I suggest we should think in terms of skills and use cases, not software components.

Reply:
I was just imagining a flat directory of tutorials in the "correct" sequence. I agree there's work to be done figuring out what needs to exist, but that's a content question (that we've already started working on) not a structure question.

Reply:
Very probably there is no flat sequence, but it's a dependency graph of skills.

I just wonder what additional information this particular list item provides. Yes, "tutorials" will contain tutorials about stuff concerning Nix-related tooling.

Reply:
Right, NixOS is both a specific way of using Nix as well as an advanced use case for Nix. In my outline from last year I lumped NixOS together with Home Manager and nix-darwin under "declarative configuration management" as they all work almost the same, but for slightly different scopes.

Reply:
That's a good idea, I'll make an update to include a section along those lines.